### DISCUSSION TITLE
Increase width of "repositories" sidebar or make view horizontally scrollable.

### DISCUSSION THREAD

**Discussion by AlanCPSC** [community/community#3984] [Other Feature Feedback, Questions, & Ideas]
As a person who constantly works with repositories that have long names. I am tired of seeing them line wrapped into incomprehensible garbage.

![Screenshot 2021-05-19 135654](https://user-images.githubusercontent.com/22120384/118883543-35b5f380-b8aa-11eb-9fe9-a93cf56618ee.png)

**Comment by PranamBhat** [comment]
Try increase/decreasing the browser zoom level or laptop/desktop screen resolution.

**Comment by ngoc-dao-cultureamp** [comment]
Related discussion:
Issue / Feature request: cannot resize files changed sidebar
https://github.com/orgs/community/discussions/86725
